lib/mozart_fetcher/http_client.ex

```diff
@@ -33,6 +33,14 @@ defmodule HTTPClient do
     end
   end
 
+  defp calculateRequestHeaders(endpoint) do
+    if String.starts_with?(endpoint, "https://fabl.api.") do
+      [{"accept-encoding", "gzip"}, {"ctx-unwrapped", "1"}]
+    else
+      [{"accept-encoding", "gzip"}]
+    end
+  end
```

Comment:
Another idiomatic approach is using pattern matching. There's a way to match the beginning of a string (but not the final part as it would be inefficient!)

```elixir
defp request_headers( "https://fabl.api." <> _) do
  [{"accept-encoding", "gzip"}, {"ctx-unwrapped", "1"}]
end

defp request_headers do
  [{"accept-encoding", "gzip"}]
end
```